lib/app/modules/stopwatch/views/stopwatch_view.dart

```diff
@@ -21,120 +21,195 @@ class StopwatchView extends GetView<StopwatchController> {
+              ),
+              ListTile(
+                onTap: () {
+                  Utils.hapticFeedback();
+                  Get.back();
+                  Get.toNamed('/settings');
+                },
+                contentPadding: const EdgeInsets.only(left: 20, right: 44),
+                title: Text(
+                  'Settings'.tr,
+                  style: Theme.of(context).textTheme.titleLarge!.copyWith(
+                        color: themeController.isLightMode.value
+                            ? kLightPrimaryTextColor.withOpacity(0.8)
+                            : kprimaryTextColor.withOpacity(0.8),
                       ),
-                    ),
-                  ),
                 ),
-              ],
-            ),
-          ),
-          const SizedBox(
-            height: 10.0,
-          ),
-          Row(
-            mainAxisAlignment: MainAxisAlignment.spaceEvenly,
-            children: [
-              FloatingActionButton(
-                heroTag: "start",
-                onPressed: controller.toggleTimer,
-                child: Obx(
-                  () => Icon(
-                    controller.isTimerPaused.value
-                        ? Icons.play_arrow_rounded
-                        : Icons.pause_rounded,
-                    size: 33,
-                  ),
+                leading: Icon(
+                  Icons.settings,
+                  size: 26,
+                  color: themeController.isLightMode.value
+                      ? kLightPrimaryTextColor.withOpacity(0.8)
+                      : kprimaryTextColor.withOpacity(0.8),
                 ),
               ),
-              // Reset button
-              FloatingActionButton(
-                heroTag: "stop",
-                onPressed: controller.resetTime,
-                child: Icon(
-                  Icons.stop_rounded,
-                  size: 33,
+              ListTile(
+                onTap: () {
+                  Utils.hapticFeedback();
+                  Get.back();
+                  Get.toNamed(Routes.ABOUT);
+                },
+                contentPadding: const EdgeInsets.only(left: 20, right: 44),
+                title: Text(
+                  'About'.tr,
+                  style: Theme.of(context).textTheme.titleLarge!.copyWith(
+                        color: themeController.isLightMode.value
+                            ? kLightPrimaryTextColor.withOpacity(0.8)
+                            : kprimaryTextColor.withOpacity(0.8),
+                      ),
+                ),
+                leading: Icon(
+                  Icons.info_outline,
+                  size: 26,
+                  color: themeController.isLightMode.value
+                      ? kLightPrimaryTextColor.withOpacity(0.8)
+                      : kprimaryTextColor.withOpacity(0.8),
                 ),
               ),
             ],
           ),
-        ],
+        ),
       ),
-      endDrawer: buildEndDrawer(context)
```

Comment:
Why was this removed?

Reply:
We're using the buildEndDrawer to keep the drawers consistent.

Reply:
i used obx for EndDrawer in line no.89 to keep drawers  consistent.

Reply:
We have a function called `buildEndDrawer` in `utils/end_drawer` which returns the endDrawer. We can reuse this function in order to make any change to the endDrawer reflect in all three pages.